### ISSUE TITLE
Strings to be translated - administration module?

### ISSUE THREAD

**Issue by horus68** [salesagility/SuiteCRM-Core#424]
Are those strings to be translated?
Should be in plain English (not merged words)... else removed from the language file!

https://github.com/salesagility/SuiteCRM-Core/blob/713b77e92fff4f9c74ffd0927c0b03bf5f2506fd/public/legacy/modules/Administration/language/en_us.lang.php#L741-L755

**Comment by pgorod** [comment]
There are more than those. They seem to be module names, and the list would serve to get singulars from plurals. It doesn't even involve the prefix `LBL_`.

I agree it shouldn't be on the language strings file (since it shouldn't be translated), but I wouldn't dare touch unless I knew which part of the code is using it. But that might prove difficult because it's likely being referenced dynamically, from a variable with a module name. So either somebody has a clue and finds the code, or I don't know how we could proceed...

Well, we could just test removing all of them and seeing if we broke any screen.
